airbyte-integrations/connectors/source-slack/source_slack/spec.json

```diff
@@ -0,0 +1,144 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/slack",
```

Comment:
We always replace the documentation_url with a generated URL on go.estuary.dev/ which we link to our docs later once we write those docs

Reply:
@mdibaiee how do I connect a new doc to https://go.estuary.dev/ and in the meantime (before the doc is written), should I point this to our main docs page?